docs/source/core_engine/01-flags.md

```diff
@@ -57,6 +57,14 @@ class MyAdapter(DiffSync):
 |---|---|---|
 | IGNORE | Do not render diffs containing this model; do not make any changes to this model when synchronizing.  Can be used to indicate a model instance that exists but should not be changed by DiffSync. | 0b1 |
 | SKIP_CHILDREN_ON_DELETE | When deleting this model, do not recursively delete its children. Can be used for the case where deletion of a model results in the automatic deletion of all its children. | 0b10 |
+| SKIP_UNMATCHED_SRC | Ignore the model if it only exists in the source/"from" DiffSync when determining diffs and syncing. If this flag is set, no new model will be created in the target/"to" DiffSync. | 0b100 |
+| SKIP_UNMATCHED_DST | Ignore the model if it only exists in the target/"to" DiffSync when determining diffs and syncing. If this flag is set, the model will not be deleted from the target/"to" DiffSync. | 0b1000 |
+| SKIP_UNMATCHED_BOTH | Convenience value combining both SKIP_UNMATCHED_SRC and SKIP_UNMATCHED_DST into a single flag | 0b1100 |
+| CRUD_NO_CREATE | Do not create and do not call create() on the DiffSyncModel during sync(), the model and the changes will still be visible in the diff. Because the destination model doesn't exist before calling create(), the flag need to be positioned on the source object. | 0b10000 |
+| CRUD_NO_UPDATE | Do not update and do not call update() on the DiffSyncModel during sync(), the model and the changes will still be visible in the diff. This flag can be positioned on both source and destination objects | 0b100000 |
+| CRUD_NO_DELETE | Do not delete and do not call delete() on the DiffSyncModel during sync(), the model and the changes will still be visible in the diff. This flag do not impact the children of this object, it's recommended to either set the flag on all objects individually or set the flag SKIP_CHILDREN_ON_DELETE | 0b1000000 |
```

Comment:
Looking at these flags side by side, it seems like there's some overlap. Is the only difference between `SKIP_UNMATCHED_DST` and `CRUD_NO_DELETE` that the former applies to both diff and sync while the latter applies only to sync? (If so, that would probably be helpful to state explicitly.) Same question for `SKIP_UNMATCHED_SRC` versus `CRUD_NO_CREATE`?

Reply:
Yes that's correct, good point
it might be best to rename these flags
- `CRUD_NO_DELETE` > `SKIP_SYNC_UNMATCHED_DST`
- `CRUD_NO_CREATE` > `SKIP_SYNC_UNMATCHED_SRC`
- `CRUD_NO_UPDATE` > `SKIP_SYNC_UPDATE`
- `CRUD_NO_UPDATE_DELETE` change to `SKIP_SYNC_ALL` and add `SKIP_SYNC_UNMATCHED_SRC` to
what do you think ? happy to use different names if you have something better

Reply:
After more discussion we decided to remove all CRUD_* flags for now.
the same behavior can be achieve by setting different flags during a diff or a sync